src/isGoodSimFcnName.m

```diff
@@ -0,0 +1,37 @@
+function goodName = isGoodSimFcnName(subsystem, simulinkFcnName)
+% isGoodSimFcnName          Checks to see if a Simulink-fcn name clashes with
```

Comment:
Please be precise and consistent with language in the comments. Please write out 'Simulink Function' instead of 'Simulink-fcn' or 'Simulink-function'. For actual parameter/variable/function names it's okay to shorten it, but be aware that the Simulink language has another block called 'Fcn'.